tests/functional/contribute/test_friends.py

```diff
@@ -4,12 +4,39 @@
 
 import pytest
 
+from selenium.common.exceptions import TimeoutException
 from pages.contribute.friends import ContributeFriendsPage
 
 
 @pytest.mark.smoke
 @pytest.mark.nondestructive
-def test_display_signup_form(base_url, selenium):
+def test_signup_default_values(base_url, selenium):
     page = ContributeFriendsPage(base_url, selenium).open()
     page.click_show_signup_form()
     assert page.is_signup_form_displayed
+    assert '' == page.email
+    assert 'United States' == page.country
+    assert page.html_format_selected
+    assert not page.text_format_selected
+    assert not page.privacy_policy_accepted
+    assert page.is_privacy_policy_link_displayed
+
+
+@pytest.mark.flaky(reruns=1)
```

Comment:
We should be able to remove the `flaky` marker now, and seeing as this won't touch ET replace it with the `nondestructive` marker.